pandas/core/categorical.py

```diff
@@ -1085,7 +1030,7 @@ def __setstate__(self, state):
 
         # Provide compatibility with pre-0.15.0 Categoricals.
         if '_categories' not in state and '_levels' in state:
-            state['_categories'] = self._validate_categories(state.pop(
```

Comment:
side issue, I think we can remove this (do we have an issue for it)?

Reply:
maybe we should create an issue on cleaning up old pickle compat (IOW we are at 0.13 now I think); we could eaily move to 0.17